enclave/sgx/revocationinfo.c

```diff
@@ -160,10 +158,12 @@ void oe_free_get_revocation_info_args(oe_get_revocation_info_args_t* args)
         free(args->tcb_issuer_chain);
 
         for (size_t i = 0; i < OE_COUNTOF(args->crl); i++)
-            free(args->crl[i]);
+            if (args->crl[i])
+                free(args->crl[i]);
```

Comment:
This was already safe per specification of standard free().  Do we know why it generated a warning?  Rather than changing all code that is already safe, it's better to have a fix that stops generating bad warnings.

Reply:
I included it because scan-build popped up many double-free warnings. The main purpose for the file edit is to add in-buffers into the free buffer section in lines 140-149 to prevent potential memory leaks.

Reply:
So if there are double free warnings, does this really solve them? `free` does not set the pointer to null, so even with this check the same double-free bug would persist.